src/compile/compile.ts

```diff
@@ -196,7 +196,7 @@ function assembleTopLevelModel(
   model: Model,
   topLevelProperties: TopLevelProperties & LayoutSizeMixins,
   datasets: Datasets = {},
-  usermeta: object
+  usermeta: Dict<any>
```

Comment:
```suggestion
  usermeta: Dict<unknown>
```

Would this still work or should this also be `object`?